### DISCUSSION TITLE
AttributeError: Can't get attribute 'C3' on <module

### DISCUSSION THREAD

**Discussion by kadirnar** [fcakyon/yolov5-pip#178] [General]
Code:
```
from yolov5 import YOLOv5

model_path = "yolov5/weights/yolov5s.pt"
device = "cuda:0" # or "cpu"

# init yolov5 model
yolov5 = YOLOv5(model_path, device)

# perform inference
results = yolov5.predict("yolov5/data/images/zidane.jpg")
results.show()
```
Error Message:
```
AttributeError: Can't get attribute 'C3' on <module 'models.common' from '.../python3.8/site-packages/models/common.py'>
```
Yolov5 Version: ```7.0.2```

**Comment by fcakyon** [comment]
this is a limitation of the original yolov5 repo. You have remove models package to use yolov5.

**Reply by kadirnar** [reply]
I'm just using the pip package. I didn't understand the reason for the error.

```
open_source
    yolov5s.pt
    terrain.jpg
    yolov5_pred.py
```
Error Message:
```
AttributeError: Can't get attribute 'C3' on <module 'models.common' from '/home/kadir/miniconda3/envs/sahi_conda/lib/python3.8/site-packages/models/common.py'>
```

**Reply by fcakyon** [reply]
You have a package named as `models` in your environment: /home/kadir/miniconda3/envs/sahi_conda/lib/python3.8/site-packages/**models**/common.py

You have to remove that package or create a fresh environment.

In Turkish:

`sahi_conda` envi icerisinde `models` adinda bir paket kurulu gorunuyor addigin hata mesajinda. yolov5'in calisabilmesi icin o paketi silmen ya da sifirdan temiz bir env olusturman gerekiyor.
